src/coreclr/hosts/unixcoreconsole/CMakeLists.txt

```diff
@@ -31,8 +31,4 @@ if(NOT CLR_CMAKE_PLATFORM_ANDROID)
   )
 endif()
 
-add_dependencies(coreconsole
-    coreclr
```

Comment:
@janvorli OK. I've fix to exclude `add_subdirectory(src/coreclr/hosts)` and `add_subdirectory(src/ildasm/unixcoreclrloader)` for NOT DEFINED CLR_CROSS_COMPONENTS_BUILD in the root CMakeFiles.txt.